core/scm.js

```diff
@@ -0,0 +1,75 @@
+'use strict';
+const Joi = require('joi');
+
+const SCHEMA_USER = Joi.object().keys({
+    id: Joi.string()
+        .required()
+        .label('Unique Identifier')
+        .example('github.com:622065'),
+
+    url: Joi.string()
+        .uri()
+        .required()
+        .label('Link to Profile')
+        .example('https://github.com/stjohnjohnson'),
+
+    name: Joi.string()
+        .required()
+        .label('Display Name')
+        .example('stjohnjohnson'),
+
+    avatar: Joi.string()
+        .uri()
+        .required()
```

Comment:
This seems specific for github. Do we know other source control would have an "avatar" for users?